kubernetes/ray-cluster.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
```

Comment:
This file isn't used for the autoscaler right? Can you make that clear in a comment?